src/clients.html

```diff
@@ -0,0 +1,164 @@
+        <p class="off-1 md:off-6 text-paragraph mb2_5 atlas md:col-6">
+          We're here to help. Over the past year, we've done significant leg work researching and understanding how we can help our clients be more responsible from both a supply chain and technology front. We're incredibly proactive in this regard! We've come to see helping our clients do better by mother nature as one of our core responsibilities.
+        </p>
+        <p class="off-1 md:off-6 text-paragraph atlas md:col-6">
+          Whether it's adding carbon offsetting at time of checkout, performing a complex lifecycle analysis for an electronics product, or powering a supply chain with mycelium based packaging, we're excited to get creative, and help our clients tell that story, too! 
+        </p>
+      </section>
+
+      <section class="full-width-content-container flex-col flex">
+        <img class="col-8 md:col-2 md:off-4 w100" src="img/greenland-b.jpg" alt="Greenland"/>
+        <p class="col-8 md:col-3 md:off-4 text-caption mt_5">
+          <span class="caption-number">(4.)</span>Greenland– <span class="color-gray-lighter">The glaciers of Greenland are also contributing to a rise in the global sea level faster than was previously believed.[86] Between 1991 and 2004, monitoring of the weather at one location (Swiss Camp) showed that the average winter temperature had risen almost 6 °C (11 °F).</span>
+        </p>
+      </section>
+
+      <section class="content-container inner-content-padding-x mxauto flex-col">
+        <h3 class="text-subtitle serif mb3 lg:col-6">Taking a hard stance.</h3>
+        <p class="off-1 md:off-6 text-paragraph atlas md:col-6">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh et in rhoncus faucibus elit purus nec. Maecenas vitae et varius ullamcorper et et. Ipsum fames facilisi posuere pellentesque.Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh et in rhoncus faucibus elit purus nec. Maecenas vitae et varius ullamcorper et et. Ipsum fames facilisi posuere pellentesque.Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh et in rhoncus faucibus elit purus nec. Maecenas vitae et varius ullamcorper et et. Ipsum fames facilisi posuere pellentesque.Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh et in rhoncus faucibus elit purus nec. Maecenas vitae et varius ullamcorper et et. Ipsum fames facilisi posuere pellentesque.
+        </p>
+      </section>
+     
+      <section class="content-container inner-content-padding-x  mxauto flex-col">
+        <p class="text-subtitle serif lg:col-8">
+          Studio Carbon Negative was initiated by Sanctuary Computer. Get in touch if you would like to know more information about Studio Carbon Negative.
+        </p>
+        <a href="https://www.sanctuary.computer/"><button class="mt2">&rarr; Visit Sanctuary Computer</button></a>
```

Comment:
These buttons inside links are weird, I would use either button or link but not both.